### ISSUE TITLE
Hover text casts shadows for each letter, this could be considered excessive and could cause unnecessary visual noise

### ISSUE THREAD

**Issue by eleriaqueen** [vE5li/korangar#41]
Hover text for npc/monsters casts shadows, for a while I thought something with the rendering of that text was wrong, until I tinkered with it and noticed the shadows. I find shadows for such small hover elements pretty unnecessary.

I had to increase the font size to be able to tell the shadow from the text.

![Screenshot_2023-07-28_10-41-21](https://github.com/vE5li/korangar/assets/4411178/2c3cff53-6b5e-4bcb-a6cc-35323895a7e0)

**Comment by hasenbanck** [comment]
@eleriaqueen Is this ticket still relevant with the new text rendering system?

**Comment by eleriaqueen** [comment]
> Is this ticket still relevant with the new text rendering system?

It doesn't look like it's still relevant, text doesn't seem to cast shadows (?)

**Comment by hasenbanck** [comment]
Yes, the hover text doesn't casts shadows now and instead is using a black outline. I will close this issue fow now then.
